pkg/collector/corechecks/net/ntp_local_defined_server_nowindows.go

```diff
@@ -0,0 +1,48 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build !windows
+
+package net
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+)
+
+func getLocalDefinedNTPServers() ([]string, error) {
+	return getNTPServersFromFiles([]string{"/etc/ntp.conf", "etc/xntp.conf"})
+}
+
+func getNTPServersFromFiles(files []string) ([]string, error) {
+	serversMap := make(map[string]bool)
+
+	for _, conf := range files {
+		content, err := ioutil.ReadFile(conf)
+		if err == nil {
+			lines := strings.Split(string(content), "\n")
+
+			for _, line := range lines {
+				line = strings.TrimSpace(line)
+				fields := strings.Fields(line)
+				if len(fields) >= 2 && fields[0] == "server" {
+					serversMap[fields[1]] = true
+				}
+			}
+		}
+	}
+
+	if len(serversMap) == 0 {
```

Comment:
You are doing this check here but not in the windows implementation. Let's align everything

Reply:
Fixed.